network-primitives/rpc_node_provisioner.tf

```diff
@@ -168,7 +168,7 @@ resource "null_resource" "start-rpc-nodes" {
       "systemctl reenable subspace.service",
 
       # set hostname
-      "hostnamectl set-hostname ${var.network-name}-rpc-node-${count.index}",
```

Comment:
Same as above

Reply:
I am trying to make the hostnames as close as possible to dns and also machine names in provider dashboard.